### PR TITLE
Hide save icons when user is not signed in

### PR DESCRIPTION
**Pull request by emilyzhaoo**
Hide save icons when user is not signed in.

When user is not signed in, save icons are not displayed. When they sign in and refresh, the save icons appear.

### CODE REVIEW

**Review by EkkiD — COMMENTED** [review]
Add some more info to the PR description with more context: https://google.github.io/eng-practices/review/developer/cl-descriptions.html

File: planet/src/main/webapp/places.js [hunk 1]
```diff
@@ -147,6 +147,7 @@ function setSearchByButton() {
 function addPlaceDetails() {
     // For each place, get the icon, name and location.
     places.forEach(function(place) {
+
```
Comment (azhang72) [inline planet/src/main/webapp/places.js]: nit: this new empty line seems random. Remove it.

File: planet/src/main/webapp/places.js [hunk 2]
```diff
@@ -343,10 +334,20 @@ function listResults() {
         div1.appendChild(div2); 
         element.appendChild(div1);
     }
-    // Add search keyword to header
+    // Add search keyword to header
```
Comment (azhang72) [inline planet/src/main/webapp/places.js]: nit: remove trailing space.

**Review by azhang72 — APPROVED** [review]
lgtm with nits pending Erick's comments.

**Review by nabil-dbz — APPROVED** [review]
LGTM!

**Pull request merged**